troubleshooting/faq.md

```diff
@@ -0,0 +1,120 @@
+# FAQ
+
+Here are answers to some frequently asked questions. If you don't see your question listed here, and it's not covered by one of the many other docs pages, feel free to [open an issue](https://github.com/npm/npme-docs/issues) and ask/propose your question there.
+
+- General
+    - [What is npm Enterprise made of?](#what-is-npme-made-of)
+    - [How do I upgrade npm Enterprise?](#how-do-i-upgrade-npme)
+    - [What is npmo/npm On-Site, and how is it related to npme/npm Enterprise?](#what-is-npmo)
+- Scopes and Packages
+    - [What's the difference between a scoped package and an unscoped package?](#difference-bw-scoped-and-unscoped)
+    - [Does using a scope make packages private automatically?](#are-scoped-packages-automatically-private)
+    - [Do I have to use a scope for the packages I publish to npme?](#do-i-have-to-use-a-scope)
+    - [Am I assigned a scope? How many scopes can I use?](#am-i-assigned-a-scope-how-many)
+    - [If I publish a package to my npm Enterprise registry, will it be published privately on the public registry too?](#are-packages-published-to-public-registry)
+
+## What is npm Enterprise made of? {#what-is-npme-made-of}
```

Comment:
Ok. But is there a good reason to prefer that over explicit, custom anchors? (Maybe I'm just being paranoid, but I like controlling the in-page links this way.)